### PR TITLE
Quality of life fixes, retranslating some strings

### PR DESCRIPTION
**Pull request by TheCrimsonDevil**

### CODE REVIEW

File: Story 1.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
   },
   "100010020130008": {
     "OriginalText": "まもなく惑星マキアの衛星軌道に到着し、\n戦闘部隊員を含む第一次探査隊が\n惑星地表へ降下します。",
-    "Text": "Once we enter Makia's orbit, the primary\nexploration team—including combat personnel—\nmay then safely descend to the surface.",
+    "Text": "Once we enter Makia's orbit, the primary\nexploration team, including combat personnel,\nmay then safely descend to the surface.",
```
Comment (TheCrimsonDevil) [inline Story 1.json]: In the patch, "-" is not a valid character, so it just combines "team" and "including" into "teamincluding".

File: other_story_strings.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "400000030100410": {
+    "OriginalText": "おっし、まずは基本の移動の確認だ。\n準備は出来てるか、新人！",
+    "Text": "First, we'll go over the movement basics.\nLet's get to it, rookie!",
```
Comment (alama) [inline other_story_strings.json]: place remove "," if there are no "Enabled" line or just add "Enabled" : false

File: other_story_strings.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "400000030100420": {
+    "OriginalText": "次は攻撃動作の確認だな。\nシミュレーターで敵を出すから\n倒していけ！",
+    "Text": "Next, we'll review attacking.\nI'm going to spawn some enemies in\nthe simulator. Defeat them!",
```
Comment (alama) [inline other_story_strings.json]: place remove "," if there are no "Enabled" line or just add "Enabled" : false

File: other_story_strings.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "400000030100430": {
+    "OriginalText": "そろそろ体が起きてきただろ？\n今度は敵の攻撃からの\n回避方法の確認をするぞ。",
+    "Text": "Has your body adjusted by now?\nI'm going to spawn some attacking\nenemies. Avoid or block their attacks.",
```
Comment (alama) [inline other_story_strings.json]: place remove "," if there are no "Enabled" line or just add "Enabled" : false

File: other_story_strings.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "400000030100440": {
+    "OriginalText": "よし、これで最後だ。\n後は好きなように敵をぶっ倒してやれ。\nもちろんアイテムの使用も自由だ。",
+    "Text": "Well, this ends the training.\nI'll spawn some more enemies, feel free to\ndefeat them any way you choose.",
```
Comment (alama) [inline other_story_strings.json]: place remove "," if there are no "Enabled" line or just add "Enabled" : false

**Pull request merged**